dfdatetime/dotnet_datetime.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+""".NET DateTime implementation."""
+
+import decimal
+
+from dfdatetime import definitions
+from dfdatetime import factory
+from dfdatetime import interface
+
+
+class DotNetDateTimeEpoch(interface.DateTimeEpoch):
+  """.NET DateTime epoch."""
+
+  def __init__(self):
+    """Initializes a .NET DateTime epoch."""
+    super(DotNetDateTimeEpoch, self).__init__(1, 1, 1)
+
+
+class DotNetDateTime(interface.DateTimeValues):
+  """.NET DateTimetimestamp
+
+  The .NET DateTime timestamp is a 64-bit signed integer that
+  stores the date and time as the number of 100 nanosecond intervls since
```

Comment:
intervls => intervals 

though I would opt to remove intervals  as in "number of 100 nanoseconds" or alternatively "100 nanosecond intervals"